### PR TITLE
CHORE: merge MyGet release into CI build

### PR DESCRIPTION
**Pull request by stijnmoreels**
We don't need a seperate duplicate of the build to release to MyGet.

 Relates to #63

### CODE REVIEW

File: build/ci-build.yml [hunk 1]
```diff
@@ -3,16 +3,23 @@ name: $(date:yyyyMMdd)$(rev:.r)
 trigger:
   branches:
     include:
-    - master
+      - master
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Is this on purpose? I don't think we need this, do we? 🤔
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example#triggers

File: build/ci-build.yml [hunk 2]
```diff
@@ -3,16 +3,23 @@ name: $(date:yyyyMMdd)$(rev:.r)
...
     include:
-    - src/*
+      - src/*
 
 pr:
   paths:
     include:
-    - src/*
-    - build/ci-build.yml
+      - src/*
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Same here

File: build/ci-build.yml [hunk 3]
```diff
@@ -3,16 +3,23 @@ name: $(date:yyyyMMdd)$(rev:.r)
 trigger:
   branches:
     include:
-    - master
+      - master
   paths:
     include:
-    - src/*
+      - src/*
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Same here

File: build/ci-build.yml (hunk 1 above)
Comment (stijnmoreels) [reply build/ci-build.yml]: Ah, no; I don't think so. We had this before on all CI builds; but as it turns out, this is triggered by default.

Comment (tomkerkhove) [reply build/ci-build.yml]: I think we actually need them but I was more referring to the additional spaces

File: build/ci-build.yml [hunk 4]
```diff
@@ -32,6 +28,35 @@ stages:
         pool:
           vmImage: 'ubuntu-16.04'
         steps:
+          - powershell: |
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Can we use this task? https://github.com/arcus-azure/azure-devops-templates/blob/master/nuget/determine-pr-version.yml

File: build/ci-build.yml [hunk 5]
```diff
@@ -1,18 +1,13 @@
 name: $(date:yyyyMMdd)$(rev:.r)
 
-trigger:
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Let's re-add this

**Pull request merged**